packages/common/lib/types/Authorization.types.ts

```diff
@@ -1,5 +1,14 @@
 import { IssuanceInitiationRequestPayload, IssuanceInitiationWithBaseUrl } from './CredentialIssuance.types';
 import { EndpointMetadata, ErrorResponse, PRE_AUTH_CODE_LITERAL } from './Generic.types';
+import {CredentialFormat} from "@sphereon/ssi-types";
+
+export interface AuthorizationDetails {
+  type: 'openid_credential' | string
+  // If the Credential Issuer metadata contains an authorization_server parameter, the authorization detail's locations common data field MUST be set to the Credential Issuer Identifier value.
+  locations?: string | string[]
```

Comment:
Since this is a property exposed to others, only use the array, as that is the only example we have right now. (Do accept a singular string on the client side though)

Reply:
So, do you want me to create another interface to be accepted as input on the client side or it should be a "DTO" to shape the output?